smartsim/_core/_cli/cli.py

```diff
@@ -67,23 +67,28 @@ def __init__(self) -> None:
             sys.exit(0)
         getattr(self, args.command)()
 
-    def build(self) -> None:
+    @staticmethod
```

Comment:
making build/clean/clobber/site() staticmethods keeps the self pointer from being passed to the Build/Clean/Clobber/Site methods. I think this is a dangerous change

Reply:
I wasn't concerned since it wasn't used, but more importantly - I have a refactor of the CLI coming next (see: https://github.com/CrayLabs/SmartSim/pull/308) and it does away with these methods altogether.